django/contrib/admin/static/admin/js/change_form.js

```diff
@@ -2,8 +2,35 @@
 {
     const inputTags = ['BUTTON', 'INPUT', 'SELECT', 'TEXTAREA'];
     const modelName = document.getElementById('django-admin-form-add-constants').dataset.modelName;
+    const buttons = document.querySelectorAll('input[type=submit]');
+    const disableSubmitButtons = () => {
+        buttons.forEach(button => {
+            button.setAttribute('disabled', true);
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute#parameters) one should set boolean attrs to the empty string:

```suggestion
            button.setAttribute('disabled', '');
```